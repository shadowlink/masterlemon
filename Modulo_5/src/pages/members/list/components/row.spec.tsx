import * as React from "react";
import { shallow } from "enzyme";
import { Row } from "./row";
import { Member } from "../viewModel";

describe("members/list/components/row tests", () => {
  it("should render row", () => {
    const member: Member = {
      avatarUrl: "url",
      id: 0,
      name: "felipe"
    };

    // Act
    const component = shallow(<Row member={member} />);

    // Assert
    expect(component).toMatchSnapshot();
  });
});
