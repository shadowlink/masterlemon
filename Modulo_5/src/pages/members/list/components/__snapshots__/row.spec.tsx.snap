// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`members/list/components/row tests should render row 1`] = `
<tr>
  <td>
    <img
      src="url"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      0
    </span>
  </td>
  <td>
    <span>
      felipe
    </span>
  </td>
</tr>
`;
